build/tools/certgen.sh

```diff
@@ -67,8 +67,9 @@ stream() {
     readonly STREAM_KEY_FILE=${certPath}/stream.key
     readonly STREAM_CSR_FILE=${certPath}/stream.csr
     readonly STREAM_CRT_FILE=${certPath}/stream.crt
-    readonly K8SCA_FILE=/etc/kubernetes/pki/ca.crt
-    readonly K8SCA_KEY_FILE=/etc/kubernetes/pki/ca.key
+
+    readonly K8SCA_FILE=${K8SCA_FILE:-/etc/kubernetes/pki/ca.crt}
+    readonly K8SCA_KEY_FILE=${K8SCA_KEY_FILE:-/etc/kubernetes/pki/ca.key}
```

Comment:
this is redundant, just readonly K8SCA_FILE=/etc/kubernetes/pki/ca.crt, K8SCA_FILE is first defined.

Reply:
If having the code, It's necessary to set K8SCA_FILE and K8SCA_KEY_FILE environment.

Reply:
WDYT @zc2638

Reply:
It makes sense, and I am using it too. 
Many private environments, their certificates are not in the default k8s path.
The script is executed by changing the required path by defining variables.
@gy95